hikari/api/chunker.py

```diff
@@ -38,14 +161,168 @@ class GuildChunker(abc.ABC):
+        timeout: builtins.int or builtins.float or builtins.None
+            The maximum amount of time the returned stream should spend waiting
+            for the next chunk event to be received before ending the iteration.
+            If `builtins.None` then this will never timeout between events.
+        limit: builtins.int or builtins.None
+            The limit for how many events the streamer should queue before
+            dropping extra received events. Leave as `builtins.None` for this to
+            be unlimited.
+        include_presences: hikari.undefined.UndefinedOr[builtins.bool]
+            If specified, whether to request presences.
+        query: builtins.str
+            If not `builtins.None`, request the members which username starts with the string.
+        query_limit: builtins.int
+            Maximum number of members to send matching the query.
+        users: hikari.undefined.UndefinedOr[typing.Sequence[hikari.snowflakes.SnowflakeishOr[hikari.users.User]]]
+            If specified, the users to request for.
+
+        !!! note
+            To request the full list of members, set `query` to `builtins.None` or `""`
+            (empty string) and `limit` to 0.
+
+        !!! note
+            The chunk request will not be sent off until the returned stream is
+            opened.
+
+        !!! warning
+            Validation errors like `builtins.ValueError` and
+            `hikari.errors.MissingIntentError` will be delayed until you open
+            the returned stream.
+
+        Returns
+        -------
+        hikari.utiltiies.event_stream.Streamer[hikari.events.shard_events.MemberChunkEvent]
+            A stream of chunk events for the generated request.
+        """
+
+    @abc.abstractmethod
+    async def get_request_status(self, nonce: str) -> typing.Optional[RequestInformation]:
+        """Return the status of a request.
+
+        Parameters
+        ----------
+        nonce: str
+            The unique identifier for the tracked request to get.
+
+        Returns
+        -------
+        typing.Optional[RequestInformation]
+            Information about the request if found, else `builtins.None`.
+        """
+
+    @abc.abstractmethod
+    async def list_requests_for_shard(
+        self, shard: typing.Union[gateway_shard.GatewayShard, int]
+    ) -> typing.Sequence[RequestInformation]:
+        """List the statuses of requests made for a specific shard.
+
+        Parameters
+        ----------
+        shard: hikari.api.shard.GatewayShard or builtins.int
+            The object or ID of the shard to get the tracked requests for.
+
+        Returns
+        -------
+        typing.Sequence[RequestInformation]
+            A sequence of data objects of information about the tracked requests
+            for the given shard.
+        """
+
+    @abc.abstractmethod
+    async def list_requests_for_guild(
+        self, guild: snowflakes.SnowflakeishOr[guilds.GatewayGuild]
```

Comment:
Is there a reason consume chunk event has a / in the signature but this does not?

Reply:
no